coderd/workspaceagentportshare.go

```diff
@@ -186,6 +187,9 @@ func convertPortShares(shares []database.WorkspaceAgentPortShare) []codersdk.Wor
 	for _, share := range shares {
 		converted = append(converted, convertPortShare(share))
 	}
+	slices.SortFunc(converted, func(i, j codersdk.WorkspaceAgentPortShare) int {
+		return (int)(i.Port - j.Port)
```

Comment:
Since I changed the ports to be uints you should probably cast before the subtract and not afterwards, like `return int(i.Port) - int(j.Port)

Reply:
These are still int32s, so I think we are fine here